build.rs

```diff
@@ -47,16 +47,20 @@ fn main() -> io::Result<()> {
     let out = PathBuf::from(env::var_os("OUT_DIR").unwrap());
 
     // Build the tries.
-    for (_, iso, _, filename, ..) in languages {
-        if cfg!(feature = "full") || (cfg!(feature = "english") && iso == "en") {
-            let path = Path::new("patterns").join(filename);
-            let tex = fs::read_to_string(&path)?;
-            let mut builder = TrieBuilder::new();
-            parse(&tex, |pat| builder.insert(pat));
-            builder.compress();
-            let trie = builder.encode();
-            fs::write(out.join(format!("{iso}.bin")), &trie)?;
+    for (name, iso, _, filename, ..) in languages {
+        let feature = name.to_lowercase();
```

Comment:
Seems like this should be [to_uppercase instead](https://doc.rust-lang.org/cargo/reference/features.html#build-scripts).

Reply:
True. In my testing, it was clearly case-insensitive, since it worked even though I did the wrong thing, but it's best to be consistent, especially if that behavior is somehow OS-specific. I'll get a fix out right away.

Reply:
Seems OS-specific then since it failed in CI and on my local computer.